firedrake/utility_meshes.py

```diff
@@ -198,6 +198,109 @@ def PeriodicUnitIntervalMesh(ncells):
     return PeriodicIntervalMesh(ncells, length=1.0)
 
 
+def OneElementThickMesh(ncells, L):
```

Comment:
Done.

On 8 June 2016 at 17:34, Lawrence Mitchell <notifications@github.com> wrote:

> In firedrake/utility_meshes.py
> <https://github.com/firedrakeproject/firedrake/pull/803#discussion_r66291049>
> :
>
> > @@ -198,6 +198,109 @@ def PeriodicUnitIntervalMesh(ncells):
> >      return PeriodicIntervalMesh(ncells, length=1.0)
> >
> >
> > +def OneElementThickMesh(ncells, L):
>
> Keep the function as is, but don't expose it in the public API. Then just
> call this guy from PeriodicRectangleMesh in the relevant case.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/firedrakeproject/firedrake/pull/803/files/0bbb07b2ed6725ca0ceca05250a2adc5380d3632#r66291049>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF0q8lljHeBYV6sbBYrj0TJIWxnGEuvgks5qJu8bgaJpZM4Iw-Dk>
> .
>



-- 
http://www.imperial.ac.uk/people/colin.cotter

www.cambridge.org/9781107663916